apps/ai-pipeline/app/pipeline/tree_builder.py

```
@@ -0,0 +1,422 @@
+        tasks = [
+            self._expand_single_branch(tree, phishing_type, difficulty, parent, choice)
+            for parent, choice in frontier
+        ]
+
+        results = await asyncio.gather(*tasks, return_exceptions=True)
+
+        next_frontier = []
+        for result in results:
+            if isinstance(result, Exception):
+                continue
+            node, new_choices = result
```

Ruff (0.15.12)

[warning] Unpacked variable `node` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)